can/util.py

```diff
@@ -279,6 +280,38 @@ def channel2int(channel: Optional[Union[typechecking.Channel]]) -> Optional[int]
     return None
 
 
+def deprecated_args_alias(**aliases):
+    """Allows to rename/deprecate a function kwarg(s) and
+    have the deprecated kwarg(s) set as alias(es)
+    """
```

Comment:
Lets add an example to the docstring:

```suggestion
def deprecated_args_alias(**aliases):
    """Allows to rename/deprecate a function kwarg(s) and
    have the deprecated kwarg(s) set as alias(es)
    
    Example:
    
        @deprecated_args_alias(oldArg="new_arg", anotherOldArg="another_new_arg")
        def library_function(new_arg, another_new_arg):
            pass
    """
```

A test would be nice, as would supporting removed arguments.

Reply:
Both are in now ;)